iStore/Cart/CartList.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CartList.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 11.03.24.
+//
+
+import SwiftUI
+
+struct CartList: View {
+    
+    var carts: [CartData]
```

Comment:
Why public?

Reply:
because if private it won't  be accessible from the cartView
![Uploading Screenshot 2024-03-11 at 4.37.33 PM.png…]()

Reply:
you can make it private with public init

Reply:
what is the difference ??

Reply:
First, to not change the value outside its class. Second to use dependency injection to make it easier for testing